src/gurt/fault_inject.c

```diff
@@ -670,4 +680,59 @@ d_should_fail(struct d_fault_attr_t *fault_attr)
 	D_SPIN_UNLOCK(&fault_attr->fa_lock);
 	return rc;
 };
+#else /* FAULT_INJECT */
+int d_fault_inject_init(void)
+{
+	D_WARN("Fault Injection not initialized feature not included in build");
```

Comment:
I found this message confusing.
```suggestion
	D_WARN("Fault Injection not initialized, feature not included in build");
```

Reply:
Let's eat grandma.  Commas save lives.